### PR TITLE
Yoda: poor man's Delete().

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich ,

Persistence for deletion is on my immediate TODO list, right after unblocking CTFO.

Thus: PTAL :-)

Thanks,
Dima

### CODE REVIEW

File: Yoda/container/dictionary/api.h [hunk 1]
```diff
@@ -164,6 +164,10 @@ struct Container<YT, Dictionary<ENTRY>> {
     }
     void Add(const std::tuple<ENTRY>& entry) { Add(std::get<0>(entry)); }
 
+    // Non-throwing deleter.
+    // TODO(dkorolev): Persist the deletion and test it.
+    void Delete(bricks::copy_free<typename YET::T_KEY> key) { mutable_.map_.erase(key); }
```
Comment (mzhurovich) [inline Yoda/container/dictionary/api.h]: There is `using` above:
`using CF = bricks::copy_free<T>;`

**Comment by mzhurovich** [comment]
@dkorolev LGTM! 
BTW, top-level method should be also added to TODO-list :)

**Comment by dkorolev** [comment]
Yes! Could you assign a bug to me pls?

On Thursday, September 10, 2015, Maxim Zhurovich <notifications@github.com>
wrote:

> @dkorolev <https://github.com/dkorolev> LGTM!
> BTW, top-level method should be also added to TODO-list :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/c5t/Current/pull/235#issuecomment-139344553>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783

**Comment by mzhurovich** [comment]
@dkorolev Done!

**Comment by dkorolev** [comment]
@mzhurovich Done! PTAL :+1:

**Pull request merged**